src/main/java/org/jenkinsci/plugins/casc/model/Sequence.java

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.plugins.casc.model;
+
+import java.util.ArrayList;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+public final class Sequence extends ArrayList<CNode> implements CNode {
```

Comment:
Would it not be better with more generic collection?

Reply:
any suggestion ?

Reply:
For yaml, I guess `ArrayList` or `List` would be fine.

Reply:
I don't get your note. The goal here is to restrict the model to only sequence of (other supported) CNode types. I selected ArrayList implementation as snakeyaml parser do create instance with a known size.

Reply:
Right, then dismiss my comment 👍